api/fittings/cors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function create() {
+  return (context, next) => {
+    const req = context.request;
+    const res = context.response;
+
+    const origin = req.headers.origin || '*';
```

Comment:
@nightsh Good job! Sure looks cleaner :sunglasses: 
But wouldn't using `*` here cause the same troubles we have now that led to #81?

From what I understand clients will have to provide an `Origin` header. Don't you think it would be better to remove  default to `*` here and inform clients to supply `Origin` by [defining it in `swagger.yml`](https://github.com/tenders-exposed/elvis-backend-node/blob/master/api/swagger/swagger.yaml#L277-L281)?